client/src/components/ItemDisplay.js

```diff
@@ -56,10 +57,24 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
+const reverseString = (string) => string.split("").reverse().join("");
+
+const addCommasToDollars = (dollars) => {
+  let dollarsString = String(dollars);
+  dollarsString = reverseString(dollarsString);
+  dollarsString = dollarsString.replace(/(.{3})/g, "$1,");
```

Comment:
what if it's over $2000? Does it this change to `$1`

Reply:
$1 is just regex for first argument, the comma that comes after is what is being inserted